boards/arm/nucleo_h743zi/nucleo_h743zi.dts

```diff
@@ -127,6 +127,12 @@ zephyr_udc0: &usbotg_fs {
 	status = "okay";
 };
 
+&dac1 {
+	status = "okay";
+	pinctrl-0 = <&dac1_out1_pa4>;
```

Comment:
Please add chosen pin to docs also for this board.

Reply:
done